aws_lambda_builders/workflows/dotnet_clipackage/utils.py

```diff
@@ -1,14 +1,107 @@
 """
 Commonly used utilities
 """
-
+import logging
 import os
 import platform
-import shutil
 import subprocess
 import zipfile
 from aws_lambda_builders.utils import which
 
+LOG = logging.getLogger(__name__)
+
+
+def _is_symlink(file_info):
```

Comment:
Can you move this into the OSUtils instead of global scope?

It would be useful to have a comment and link to the code in SAM CLI. Just helps future people know we copied it from one place to the other.

Reply:
Yeah, I assume you want all the new methods inside OSUtils then? Is that right? I'll add the comment and a link to the code in SAM CLI